apps/server/.env.example

```diff
@@ -1,25 +1,29 @@
- # Env: The mocked environment will mock out your https calls as defined in mock.js
+# Env: The mocked environment will mock out your https calls as defined in mock.js
 Env=pre
 
-SophtronClientId="" # sophtron.com -> api settings -> direct auth -> UserId
-SophtronClientSecret="" # sophtron.com -> api settings -> direct auth -> AccessKey
```

Comment:
I looked through the code, and these variables are not used anywhere, and are duplicates of the other "SOPHTRON*" variables.

Reply:
Good find